index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <img class="hamburger" src="images/menu.svg" alt="menu">
+    </div>
+  </header>
+  <section class="section-text">
+    <div class="section-title"><h1>I'm Daniel <br>Glad to see <br>you!</h1></div>
```

Comment:
- [ ] i'm daniel should be in an `h1 tag` while glad to see you is an `h2.` please fix that